test/api_test.py

```diff
@@ -135,3 +135,14 @@ def test_create_command(self):
 
     self.assertIsInstance(custom_command, CustomCommand)
     self.assertEqual(custom_command.command, 'helloWorld')
+
+  def test_convert_units(self):
+    """
+    Converting between iota standard units.
+    """
+    api = Iota(MockAdapter())
+
+    self.assertEqual(api.convert_units('1.618 Mi', 'i'), 1618000)
+    self.assertEqual(api.convert_units('42 i', 'Ki'), 0.042)
+    with self.assertRaises(ValueError):
+      api.convert_units('3.141596 Xi', 'Pi')
```

Comment:
OK, I've managed to at least run all tests in `transaction.py` with `python setup.py test -s test.transaction_test`
Not getting it to run `test.transaction_test.UnitConverterTestCase` only, though. Unit test runs but reports: `Ran 0 tests in 0.001s`